system/wymeditor/plugins/image_upload/jquery.wymeditor.image_upload.js

```diff
@@ -45,12 +45,37 @@ WYMeditor.editor.prototype.image_upload = function() {
 				} else {
 					jQuery(".wym_src", doc).val(response.thumbUrl);
 					jQuery(".wym_alt", doc).val(response.original_filename);
+					jQuery(".wym_attribution", doc).val(response.attribution);
```

Comment:
IIUC this processes an "attribution" attribute returned from the image upload handler, but I see no changes in the upload handler?

Reply:
True! A next feature will be that the upload will return "attribution". This code is unaware of the state of the upload and already prepared.